MovieHubRx/Scenes/MovieDetail/MovieDetailInfoCell/MovieDetailInfoCell.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  MovieDetailInfoCell.swift
+//  MovieHubRx
+//
+//  Created by Duy Nguyễn on 15/05/2024.
+//
+
+import UIKit
+import Reusable
+
+final class MovieDetailInfoCell: UITableViewCell, NibReusable {
+
+    @IBOutlet private weak var containerView: UIView!
+    @IBOutlet private weak var backDropImageView: UIImageView!
+    @IBOutlet private weak var posterImageView: UIImageView!
+    @IBOutlet private weak var playButton: UIButton!
+    @IBOutlet private weak var movieNameLabel: UILabel!
+    @IBOutlet private weak var timeLabel: UILabel!
+    @IBOutlet private weak var favoriteButton: UIButton!
+    @IBOutlet private weak var overViewLabel: UILabel!
+    @IBOutlet private weak var descriptionLabel: UILabel!
+    @IBOutlet private weak var ratingLabel: UILabel!
+    
+    var tappedFavorite: ((Bool) -> Void)?
+    var tappedPlay: ((String?) -> Void)?
+    private var isFavorited = false
+    private var key: String?
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        configView()
+    }
+    
+    private func configView() {
+        backDropImageView.addGradientOverlay()
+        movieNameLabel.textColor = .white
+        descriptionLabel.textColor = .secondaryLabel
+        overViewLabel.text = "movie.overview".localized()
+        favoriteButton.tintColor = .systemPink
+        
+        posterImageView.do {
+            $0.layer.cornerRadius = Constants.cornerImage
+            $0.layer.cornerCurve = .continuous
+            $0.layer.borderColor = UIColor.white.cgColor
+            $0.layer.borderWidth = 3
+        }
+        
+        let configuration = UIImage.SymbolConfiguration(pointSize: 40)
+        playButton.do {
+            $0.setPreferredSymbolConfiguration(configuration, forImageIn: .normal)
+            $0.tintColor = .white
+        }
+    }
+    
+    func bind(movie: Movie, favoriteStatus: Bool) {
+        isFavorited = favoriteStatus
+        descriptionLabel.text = movie.overview
+        movieNameLabel.text = movie.title
+        timeLabel.text = String(format: "common.runtime: %@ minutes".localized(),
+                                "\(movie.runtime )")
+        let voteLabel = (movie.voteAverage ).toStringRating()
+        ratingLabel.text = voteLabel
+        
+        let backDropUrl = APIUrls.Endpoint.image(urlString: movie.backDropPath).url
+        backDropImageView.sd_setImage(with: URL(string: backDropUrl))
+        let posterUrl = APIUrls.Endpoint.image(urlString: movie.poster).url
+        posterImageView.sd_setImage(with: URL(string: posterUrl))
+        key = movie.videos.results.first?.key
+        updateFavoriteButtonImage(isFavorited: isFavorited)
+    }
+    
+    func updateFavoriteButtonImage(isFavorited: Bool) {
+        favoriteButton.tintColor = isFavorited ? .systemPink : .white
+    }
+    
+    @IBAction func tappedFavoriteButton(_ sender: Any) {
```

Comment:
tương tự cái dưới, cơ mà tìm hiểu sử dụng bằng rx thử e
```suggestion
    @IBAction 
    private func tappedFavoriteButton(_ sender: Any) {
```